src/test/java/com/google/opensesame/mockObjects/MockGHObjects.java

```diff
@@ -0,0 +1,109 @@
+package com.google.opensesame.mockObjects;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+
+public class MockGHObjects {
+
+  // #### MOCK GHUser ####
+
+  /** The name of the mocked GHUser returned by setupMockUser() */
+  public static String MOCK_GHUSER_NAME = "MOCK_NAME";
+  /** The bio of the mocked GHUser returned by setupMockUser() */
+  public static String MOCK_BIO = "MOCK_BIO";
+  /** The Avatar URL of the mocked GHUser returned by setupMockUser() */
+  public static String MOCK_AVATAR_URL = "MOCK_AVATAR_URL";
+  /** The bio of the mocked GHUser returned by setupMockUser() */
+  public static String MOCK_LOCTAION = "MOCK_LOCTAION";
+
+  /**
+   * The HTML URL of the mocked GHUser returned by setupMockUser()
+   *
+   * @throws MalformedURLException
+   */
+  public static URL MOCK_HTML_URL() throws MalformedURLException {
+    return new URL("http://localhost/");
+  }
+
+  /**
+   * Creates an instance of a GHUser mocked with Mockito. The mocked user comes with the following
+   * functions predefined;
+   * <li>getBio() -> MOCK_BIO
+   * <li>getAvatarUrl() -> MOCK_AVATAR_URL
+   * <li>getLocation() -> MOCK_LOCATION
+   * <li>getName() -> MOCK_NAME
+   * <li>getHtmlUrl().toString() -> MOCK_HTML_URL();
+   *
+   *     <p>NOTE: all functions return fixtures defined in this class. MOCK_HTML_URL() is a function
+   *     due to complications with Java's URL Class.
+   *
+   * @return a mocked instance of a GHUser
+   * @throws IOException
+   */
+  public static GHUser setupMockUser() throws IOException {
```

Comment:
mega nit: `setupMockUser()` doesn't immediately make me think that something is being returned. Consider `createMockUser()` and `createMockRepository()` for this function and the function below:

```suggestion
  public static GHUser createMockUser() throws IOException {
```